src/command_modules/azure-cli-managementpartner/HISTORY.rst

```diff
@@ -0,0 +1,12 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.1
```

Comment:
You don't need this.  If it's your initial release, you should just have the single entry.